src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -350,36 +363,52 @@ public SslProtocols SslProtocols
 
         public HttpClientHandler()
         {
-            _rtFilter = new RTHttpBaseProtocolFilter();
-            _handlerToFilter = new HttpHandlerToFilter(_rtFilter);
+            _rtFilter = CreateFilter();
+            _rtFilterWithNoCredentials = CreateFilter();
+
+            _handlerToFilter = new HttpHandlerToFilter(_rtFilter, _rtFilterWithNoCredentials, this);
             _handlerToFilter.RequestMessageLookupKey = RequestMessageLookupKey;
             _handlerToFilter.SavedExceptionDispatchInfoLookupKey = SavedExceptionDispatchInfoLookupKey;
             _diagnosticsPipeline = new DiagnosticsHandler(_handlerToFilter);
 
             _clientCertificateOptions = ClientCertificateOption.Manual;
 
+            _useCookies = true; // deal with cookies by default.
+            _cookieContainer = new CookieContainer(); // default container used for dealing with auto-cookies.
+
+            _allowAutoRedirect = true;
+            _maxAutomaticRedirections = 50;
+
```

Comment:
Originally hard-coded to 10, with a comment about the WinRT implementation via wininet. Why not 10 here?